web3/auto/infura/loader.py

```diff
@@ -1,24 +1,17 @@
 import logging
 import os
 
-from web3 import (
-    HTTPProvider,
-    Web3,
-)
-
 INFURA_MAINNET_BASE_URL = 'https://mainnet.infura.io'
+INFURA_ROPSTEN_BASE_URL = 'https://ropsten.infura.io'
 
 
-def load_infura_url():
+def load_infura_url(base_url):
```

Comment:
*nitpick*

`load` feels out of place here.  I would expect load to be for things like loading from a file.  Maybe `get_` or `construct_` or `build_`?  Not convinced these are much better though.